### PR TITLE
Add page for the Community Github Org with list of repos

### PR DESCRIPTION
**Pull request by matt-oakes**
# What's in this PR?

* A script to fetch the list of repos from the Community Github Org and then parse out the information that we need.
* An example page to display the repo information.
* Run the scripts when the CI server, and Netlify, builds the website.

# Still to do
This PR is meant to start a conversation about what we would like to include on the community page of the website. There are still lots of open questions:

## What do we want to include on the community page?
Currently, I have just included the first paragraph of the [meta repo](https://github.com/react-native-community/meta) and a bare bones repo list. There is a lot more we could include:

* Highlight the key repos like "releases", "discussions-and-proposals", and "meta".
* Include some information about why the org exists and what the difference between the libraries contained in it and from other community members is.
* Highlights of the guidelines we impose and why it matters (consistent code standards and well tested).

## How do we make the repo list nicer?
Currently, I have just used the styling of the "showcase" to make a grid and included the repo name and the description. Other information we could include is:

* Number of stars.
* How "standards-compliant" the repo is.
* How active the repo is.
* How many downloads it has had.

## Where should this page live in the navigation structure?
There is currently a "community" link in the header, however, this goes to the "help" page which then includes help links and links to 3rd-party communities. This is useful information so we should keep it.

To me, it makes sense to have the new Community page go to the new page in this PR and have the current Help page accessible under a new "Help" link in the header.

## Anything else?
This is meant to start a conversation, so... GO!

### CODE REVIEW

**Comment by matt-oakes** [comment]
⬆️ That's likely because I've added a new script to the preview script which required a `GITHUB_TOKEN` environment variable. This is used to get the community repos from the Github API on each deploy.

File: website/pages/en/community.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      <div className="pageContainer">
+        <Container padding={['bottom']}>
+          <div className="communitySection">
+            <div className="prose">
+              <h1>React Native Community Github Organisation</h1>
+              <p>
+                React-Native-Community (RNC) is a GitHub organization dedicated
+                to the collaborative and community-driven long-term maintenance
+                of React Native packages &amp; libraries - this is also made
+                possible by collaboration with the Facebook team in charge for
```
Comment (bartolkaruza) [inline website/pages/en/community.js]: "in charge of"

**Comment by grabbou** [comment]
Great work! Number of issues / opened PRs might also be nice, but I think this is something we can configure later too.

File: website/pages/en/community.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/**
```
Comment (grabbou) [inline website/pages/en/community.js]: There's already "Community" page -> https://facebook.github.io/react-native/help (the slug is confusing, it's named Community from the top bar). Maybe we could use that one somehow?

**Comment by cpojer** [comment]
This is amazing.

File: website/pages/en/community.js (hunk 2 above)
Comment (matt-oakes) [reply website/pages/en/community.js]: I put this in the PR description:

> There is currently a "community" link in the header, however, this goes to the "help" page which then includes help links and links to 3rd-party communities. This is useful information so we should keep it.
>
> To me, it makes sense to have the new Community page go to the new page in this PR and have the current Help page accessible under a new "Help" link in the header.

I don't really mind though and I'm happy to go with whatever people think on the website structure.

**Comment by matt-oakes** [comment]
@cpojer @grabbou Should I modify this PR to only include the data fetching part for now and then we can follow it up with another PR to actually make use of the data? I don't feel like I'm the best person to be making decisions about the website structure on my own but this PR can be a step towards it.

Let me know if that works for you?

**Comment by cpojer** [comment]
Yeah that works for me :)

**Comment by matt-oakes** [comment]
Thanks for the reviews everyone!

I have removed the page I created and left this PR to just be the sync script. Someone will need to check that that CircleCI and Netlify has the correct `GITHUB_TOKEN` environment variable before this gets merged.

The next step will be for someone to make use of the new data in a community page :+1:

**Comment by matt-oakes** [comment]
I forgot to say, I also added open issues and PRs to the data 👍

**Comment by charpeni** [comment]
Thanks for doing this!

I just configured Netlify with a `GITHUB_TOKEN`, we'll be able to play with it.

**Comment by matt-oakes** [comment]
Perfect. Thanks!

**Comment by cpojer** [comment]
What's the status of this? Where can I check out what this looks like? I'd like to get this in :)

**Comment by matt-oakes** [comment]
@cpojer I removed all of the visible changes from this PR as there was no consensus on how the data should be displayed. It now just adds the data loading logic. It's ready to merge, but you'll need another PR to display the data somewhere.

**Comment by rachelnabors** [comment]
@matt-oakes I'm interested in picking this up. Are you available to collaborate?

**Pull request merged**